src/Twig/SecurityPolicyContainerAware.php

```diff
@@ -28,42 +28,42 @@
  * @author fabien.potencier@symfony.com
  * @author thomas.rabaix@sonata-project.org
  */
-class SecurityPolicyContainerAware implements SecurityPolicyInterface
+final class SecurityPolicyContainerAware implements SecurityPolicyInterface
 {
     /**
      * @var string[]
      */
-    protected $allowedTags;
+    private $allowedTags;
```

Comment:
imo you should name the protected -> private move in the `UPGRADE.md`, too

Reply:
well, the class is final so they shouldn't really care?